### PR TITLE
Release of API version 1.0.0-rc.1 (don't merge, to be closed)

### PR DESCRIPTION
**Pull request by bigludo7**
#### What type of PR is this?

Add one of the following kinds:

* documentation
* subproject management

Note:  this PR has exactly same contents than #121 that was closed because of branch issue

#### What this PR does / why we need it:

Generation of first release-candidate for NumberVerification API

This first release candidate r1.1 contains the definition and documentation of the release-candidate of the NumberVerification API v1.0.0-rc.1.

#### Which issue(s) this PR fixes:

Fixes #

#### Changelog input

```
 release-note

```

#### Additional documentation 

This section can be blank.

```
docs

```

### CODE REVIEW

**Review by fernandopradocabrillo — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,50 +1,88 @@
...
+**number-verification 1.0.0-rc.1** is the first release-candidate version for the v1.0.0 of the number verification API.
+
+- API definition **with inline documentation**:
+    - OpenAPI [YAML spec file](https://github.com/camaraproject/NumberVerification/blob/r1.1/code/API_definitions/number_verification.yaml)
+    - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/NumberVerification/blob/r1.1/code/API_definitions/number_verification.yaml&nocors)
+    - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/NumberVerification/blob/r1.1/code/API_definitions/number_verification.yaml)
+    
+**number-verification 1.0.0-rc.1 is the first stable version for CAMARA Number Verification API**
+
+This version contains significant changes compared to v0.3.1, and it is not backward compatible.
```
Comment (hdamker) [inline CHANGELOG.md:40]: Same changes as in https://github.com/camaraproject/OTPValidation/pull/74:

```suggestion
**number-verification 1.0.0-rc.1** is the first release-candidate version for the v1.0.0 of the number-verification API.
This version contains significant changes compared to v0.3.1, and it is not backward compatible.

- API definition **with inline documentation**:
    - OpenAPI [YAML spec file](https://github.com/camaraproject/NumberVerification/blob/r1.1/code/API_definitions/number_verification.yaml)
    - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/NumberVerification/blob/r1.1/code/API_definitions/number_verification.yaml&nocors)
    - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/NumberVerification/blob/r1.1/code/API_definitions/number_verification.yaml)

```

File: code/API_definitions/number_verification.yaml
Comment (hdamker) [inline code/API_definitions/number_verification.yaml:1]: number_verification.yaml has been uploaded as a new file, GitHub is showing that all lines were replaced, which makes it difficult to see the actual changes.

**Comment by hdamker** [comment]
number_verification.yaml seems to been uploaded as a new file in https://github.com/camaraproject/NumberVerification/pull/123/commits/435b7bdfea915519a9d1510bcc3cb61b9482b443. 

GitHub is showing that all lines were replaced, which makes it difficult to see the actual changes, especially the ones from the following commits here in the PR.

**Comment by fernandopradocabrillo** [comment]
> number_verification.yaml seems to been uploaded as a new file in [435b7bd](https://github.com/camaraproject/NumberVerification/commit/435b7bdfea915519a9d1510bcc3cb61b9482b443).
> 
> GitHub is showing that all lines were replaced, which makes it difficult to see the actual changes, especially the ones from the following commits here in the PR.

Hi @hdamker, I'm trying to fix this but I can't commit into the PR. It seems that it's locked. As admin, do you have permissions to unlock it? Converting it to draft doesn't seem to work

**Comment by hdamker** [comment]
@fernandopradocabrillo I have no clue why this PR should have been locked, but I thought it is anyway easier to recreate the originally PR #121, but just against the correct base branch (i.e. `main`). Therefore I recreated the branch https://github.com/camaraproject/NumberVerification/tree/bigludo7-patch-2 and used it to open the new PR #126. 
(It would have even been possible to just edit the base path in PR #121, but not after it was closed and I deleted the previous base path).

What is left do do is to redo the following commits:

https://github.com/camaraproject/NumberVerification/pull/123/commits/83fd1a22e4bba08c068b3fa2b654ba0d4138d37e, https://github.com/camaraproject/NumberVerification/pull/123/commits/65b1db0d15582f041a2eea68006bbe46ec71fd3f, and https://github.com/camaraproject/NumberVerification/pull/123/commits/73ee72f9468ef66a5343c65d1c8179c4b3e804b2

But I would recommend to do them not in #126 but in a separate PR before the #126 get's merged and the release done.

**Comment by hdamker** [comment]
Closing it and deleting the branch

**Pull request closed without merging**